magic/rotation.py

```diff
@@ -196,6 +196,11 @@ def last_run_time() -> Optional[datetime.datetime]:
         return None
 
 def read_rotation_files() -> Tuple[int, int, List[Card]]:
+    runs_str = redis.get_str('decksite:rotation:summary:runs')
+    runs_percent_str = redis.get_str('decksite:rotation:summary:runs_percent')
+    cards = redis.get_list('decksite:rotation:summary:cards')
+    if runs_str is not None and runs_percent_str is not None and cards is not None:
+        return int(runs_str), int(runs_percent_str), [Card(c, predetermined_values=True) for c in cards]
```

Comment:
Can you use `redis.get_container` here instead of introducing the `predetermined_values` var? Even if you had to make a new constructor for Card that takes a Container/Dict (which probably already exists) it seems better?

Reply:
Hmm, let me think about it.